css/styles.css

```diff
@@ -188,3 +188,51 @@ p {
 .trash-taste {
   cursor: pointer;
 }
+
+.module-container {
+  position: absolute;
+  width: 100%;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
+
+.module {
+  width: 25%;
+  height: 25vh;
+  background-color: #7c7461;
+  z-index: 999;
+  border-radius: 5px;
+  box-shadow: 0 1px 10px black;
+}
+
+.inside-module {
+  width: 100%;
+  padding: 0 4rem;
+  line-height: 1.5rem;
+  margin: 1.5rem 0 0;
+}
+
+.justify-content-space-around {
+  justify-content: space-around;
+}
+
+.cancel {
+  width: 35%;
```

Comment:
For buttons, it's better to leave their width alone and instead add padding. Or, you can give them a more specific width in `px`, but percentage width looks weird usually.